unlock-js/src/__tests__/v10/web3Service.test.js

```diff
@@ -324,9 +324,10 @@ describe('Web3Service', () => {
           ],
         }
 
-        web3Service.on('transaction.updated', (tHash, lock) => {
+        web3Service.on('transaction.updated', (tHash, transactionUpdate) => {
           expect(tHash).toBe('hash')
-          expect(lock).toEqual({
+          expect(transactionUpdate).toEqual({
+            for: checksumLockAddress,
```

Comment:
This is probably just a naming thing, but is `for` the LockAddress or is it for the recipient - the address that will receive the key?

Reply:
the recipient, as in "this key is for Joe"
The lock is the "to" address, the purchaser is the "from" address. What do you think about naming now that you have this context?

Reply:
ok, I just pushed an update to fix the `checksumLockAddress` to be `owner` instead